tracks/cpp/exercises/grains/mentoring.md

```diff
@@ -0,0 +1,45 @@
+# Mentoring
+
+## First function
+
+The first way to implement this function is to use a for loop :
+```cpp
+unsigned long long square(int indice) {
```

Comment:
Here and later: "indice" sounds like French or Italian, right? Since everything else is in English how about `index`?

Reply:
Yes I'm french, it happens sometimes that I let a french word in my code. I will do this changes thx :)